src/views/docs-view/utils/__tests__/all-docs-server.test.ts

```diff
@@ -0,0 +1,96 @@
+import { RootDocsPath } from 'types/products'
+import {
+	prefixAllDocsSubpaths,
+	removeCustomLandingPaths,
+	parseRootDocsPath,
+} from '../all-docs-server'
+
+describe('prefixAllDocsSubpaths', () => {
```

Comment:
Added some basic tests here, not exactly exhaustive but figure it's better than nothing.